### DISCUSSION TITLE
Custom roles

### DISCUSSION THREAD

**Discussion by pigeonflight** [Budibase/budibase#3762] [Help] [answered]
Is it possible to create a custom role?
For example, I may want to have a role for "can submit forms" different from "can receive alerts".

**Comment by mjashanks** [comment, marked as answer]
Yes - you can create custom roles. Go to the `Users` table, and click `Edit Roles`.

Note that a user can only have one role in a single app - i.e. they cannot have `can submit forms` AND `can receive alerts` - you'll need to construct your roles for each combination of permissions that you need.

**Reply by pigeonflight** [reply]
@mjashanks This is what I see when I log in
![image](https://user-images.githubusercontent.com/31827/145895719-f791ae04-6d73-4e43-a216-2bce84d1c11a.png)

(do I not have enough permissions to add a role?) 
I have development and admin access.

**Reply by pigeonflight** [reply]
Just a note, since it is now clear to me. The critical thing is, as @mjashanks said, roles are bound to apps. You can't add new global roles, they are at the application level.
